src/extension/AirGapSigner.ts

```diff
@@ -33,56 +35,128 @@ export class AirGapOperationProvider implements OperationProvider {
     return forgedTx.binaryTransaction
   }
 
+  public async operationGroupFromWrappedOperation(
```

Comment:
Would it make sense to move those methods to the coinlib so we can also use them in airgap?

Reply:
it's fine for now